web/src/site-admin/externalServices.ts

```diff
@@ -60,6 +60,17 @@ export const ALL_EXTERNAL_SERVICES: ExternalServiceMetadata[] = [
   "url": "https://phabricator.example.com",
   "token": "",
   "repos": []
+}`,
+    },
+    {
+        kind: GQL.ExternalServiceKind.OTHER,
+        displayName: 'Other',
+        defaultConfig: `{
+  // You can define the base "url" and set "repos" as paths only (e.g. my/repo), or
+  // leave "url" unset and define each repo in "repos" with an absolute URL.
+  // Supported URL schemes are: http, https, git and ssh
+  "url": "https://ACCESSTOKEN@my-other-githost.example.com",
```

Comment:
After testing it myself, I think we should even remove this accesstoken because we don't want to imply that will work with any arbitrary code host.

```suggestion
  "url": "https://my-other-githost.example.com",
```